Makefile

```
@@ -293,3 +293,22 @@ docs-serve-dev:
 	  -v "$(CURDIR):/k0s:ro" \
 	  -p '$(DOCS_DEV_PORT):8000' \
 	  k0sdocs.docker-image.serve-dev
+
+spdx/spdx.json:
+	docker run --rm \
+	  -v "$(CURDIR)/go.mod:/k0s/go.mod" \
+	  -v "$(CURDIR)/sbom:/out" \
+	  anchore/syft:v0.90.0 \
+	  /k0s -o spdx-json@2.2=/out/spdx.json
```

So that the generated file has the right permissions:

```suggestion
	mkdir -p -- '$(dir $@)'
	docker run --rm \
	  -v "$(CURDIR)/go.mod:/k0s/go.mod" \
	  -v "$(CURDIR)/sbom:/out" \
	  --user $(BUILD_UID):$(BUILD_GID) \
	  anchore/syft:v0.90.0 \
	  /k0s -o spdx-json@2.2=/out/spdx.json
```